test/test_tensorexpr_pybind.py

```diff
@@ -40,6 +41,109 @@ def compute(i):
+            out = torch.zeros_like(ref)
+            codegen.call([image.float(), weight.float(), out.float()])
+            torch.testing.assert_allclose(out, ref)
+
+    @unittest.skipIf(not LLVM_ENABLED, "LLVM backend not enabled")
+    def test_conv_with_loops(self):
+        def loop(v, bound, body):
+            return torch._C._te.For.make(v, torch._C._te.ExprHandle.int(0), bound, body)
+        with kernel_arena_scope():
+            N, C, H, W = 1, 8, 6, 6 # Image: batch size = N, channel num = C, height = H, width = W
+            K, R, S = 8, 3, 3 # kernel: output channel num = K, kernel height = R, Kernel width = S
+            Pad = 1 # Padding
+
+            OH, OW = H - R + 1, W - S + 1
+
+            (NN, KK, HH, WW) = [torch._C._te.ExprHandle.int(x) for x in [N, K, OH, OW]] # result tensor shape
+            (CC, RR, SS) = [torch._C._te.ExprHandle.int(x) for x in [C, R, S]]
+            (HI, WI) = [torch._C._te.ExprHandle.int(x) for x in [H, W]]
+            dtype = torch._C._te.Dtype.Float
+
+            def get_dim_args(dims):
+                dim_args = []
+                for dim in dims:
+                    dim_args.append(torch._C._te.DimArg(dim, 'i' + str(len(dim_args))))
+                return dim_args
+
+            Pimage = torch._C._te.Placeholder('image', dtype, [NN, CC, HI, WI])
+            Pweight = torch._C._te.Placeholder('weight', dtype, [KK, CC, RR, SS])
+            Pout = torch._C._te.Placeholder('conv', dtype, [NN, KK, HH, WW])
+
+            dint = torch._C._te.Dtype.Int
+            n, k, h, w, c, r, s = [torch._C._te.VarHandle(s, dint) for s in ["n", "k", "h", "w", "c", "r", "s"]]
+            stmt = loop(n, NN,
+                    loop(k, KK,
+                        loop(h, HH,
+                            loop(w, WW,
+                                loop(c, CC,
+                                    loop(r, RR,
+                                        loop(s, SS,
+                                            Pout.store(
+                                                [n, k, h, w],
+                                                Pout.load([n, k, h, w]) +
+                                                Pimage.load([n, c, h+r, w+s]) *
+                                                Pweight.load([k, c, r, s])
+                                                )))))))
+                    )
+
+            loopnest = torch._C._te.LoopNest(stmt, [Pout.buf()])
+            loopnest.prepare_for_codegen()
+            stmt = torch._C._te.simplify(loopnest.root_stmt())
+            codegen = torch._C._te.construct_codegen('llvm', stmt, [torch._C._te.BufferArg(x) for x in [Pimage, Pweight, Pout]])
+
+            image = torch.ones(N, C, H, W)
+            weight = torch.ones(K, C, R, S)
+            ref= F.conv2d(image, weight)
+            out = torch.zeros_like(ref)
+            codegen.call([image.float(), weight.float(), out.float()])
+            torch.testing.assert_allclose(out, ref)
+
```

Comment:
Not sure if it's proper to add the tests here. We already have a conv test written in cpp. @ZolotukhinM